float64[6] v
